sfaira/train/summaries.py

```diff
@@ -1162,19 +1176,20 @@ def __init__(
 
     def create_summary_tab(self):
         metrics = list(self.evals.values())[0]['val'].keys()
+        run_ids = [run_id.replace('-', '_') for run_id in self.run_ids]
```

Comment:
@davidsebfischer a little hacky but I guess this would work to handle the new model_id format. what do you think?